src/run.py

```diff
@@ -677,22 +682,13 @@ def runner(mode, outdir, alt_cache, logger, additional_bind_paths = None,
     if temp not in additional_bind_paths.split(','):
         addpaths.append(temp)
     bindpaths = ','.join(addpaths)
-
+    
     # Set ENV variable 'SINGULARITY_CACHEDIR' 
     # to output directory
     my_env = {}; my_env.update(os.environ)
     cache = os.path.join(outdir, ".singularity")
     my_env['SINGULARITY_CACHEDIR'] = cache
     my_env['APPTAINER_CACHEDIR'] = cache
-    # Removing R_SITE_LIB environment variable
-    # due to issue: https://github.com/OpenOmics/chrom-seek/issues/28
-    # Using SINGULARITY_CONTAINALL or APPTAINER_CONTAINALL
-    # causes downstream using where $SLURM_JOBID is 
-    # NOT exported within a container.
-    if 'R_LIBS_SITE' in my_env:
```

Comment:
Are we still relying on any `module load R` for any rules? If so, one thing that we should also do is unset `R_LIBS_USER` in any rule's shell commands that are using a module load verison of R.

Reply:
![image](https://github.com/user-attachments/assets/57516301-e21d-439d-b953-321b5a16c607)

Reply:
When R is module loaded, this environment variable is set, so even if you are unsetting this variable right before submission of the master job, if the rule module loads R, it will be reset. To handle this, we will need to unset that variable at the rule-level in the shell command right before we run anything R related.